### PR TITLE
css-three-column-layout - Alvin Cho

### PR DESCRIPTION
**Pull request by c-alvin**
image

### CODE REVIEW

File: css-three-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+.text-center {
+  text-align: center;
+}
+
+.div1 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css:68]: This is a pretty useless class name. What is this for?

File: css-three-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+.padding250 {
+  padding: 0 250px;
+}
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css:72]: _**Yikes**_, that's a lot of padding. As we discussed, you can likely address this by making the `max-width` of your `container` class lower.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by c-alvin** [comment]
image

**Review by codingcodymiller — APPROVED** [review]
Nice work!

**Pull request merged**